cloudformation_seed/cfn_stack.py

```diff
@@ -42,17 +43,39 @@ def get_stack_output(self, output_name: str) -> Optional[str]:
                 log.debug(f'Output {self.stack_name}.{output_name} = {xo["OutputValue"]}')
                 return xo['OutputValue']
 
+    def format_tags(self, tags_passed):
+        self.stack_tags = [{'Key': k, 'Value': str(v)} for k, v in tags_passed.items() if v is not None]
+        print(self.stack_tags)
+
+    def validate_tags(self, tags_passed):
+        for k, v in tags_passed.items():
+            if len(k) > 129:
+                raise RuntimeError('Tag Key {0} cannot be more than 128 characters long'.format(k))
+            if len(v) > 257:
```

Comment:
> `> 255`?

same as above, i'' change it to ``` > 256 ```


> Also `< 1`? Because `huh = {'': 'what?'}` is valid?

I've just tested deploying a stack with a tag that has no key but the mapping doesn't get parsed as its an invalid key.
![image](https://user-images.githubusercontent.com/40017704/59073709-8cb79d80-890b-11e9-8f3e-6111960983a4.png)

So it seems to be handled already unless you want me to capture that error and display a different one?

Reply:
There's no need to safetynet everything, the default error actually makes sense.